buf-connect-go.gen.yaml

```diff
@@ -0,0 +1,8 @@
+version: v1
+plugins:
+  - name: go
+    path: lib/prehog/bin/protoc-gen-go
```

Comment:
Not for this PR, but we should consolidate so we don't have tooling versions tracked by go.mod and tooling versions tracked via buildbox.